packages/api/tests/models/test-granules-model.js

```diff
@@ -5,23 +5,48 @@ const test = require('ava');
 const sinon = require('sinon');
 
 const aws = require('@cumulus/common/aws');
+const { constructCollectionId } = require('@cumulus/common/collection-config-store');
 const ingestAws = require('@cumulus/ingest/aws');
 const launchpad = require('@cumulus/common/launchpad');
 const StepFunctions = require('@cumulus/common/StepFunctions');
 const { randomString } = require('@cumulus/common/test-utils');
+const { removeNilProperties } = require('@cumulus/common/util');
 const cmrjs = require('@cumulus/cmrjs');
 const { CMR } = require('@cumulus/cmr-client');
 const { DefaultProvider } = require('@cumulus/common/key-pair-provider');
 
 const range = require('lodash.range');
 
 const { Granule } = require('../../models');
+const { filterDatabaseProperties } = require('../../lib/FileUtils');
 const { fakeFileFactory, fakeGranuleFactoryV2 } = require('../../lib/testUtils');
+const { deconstructCollectionId } = require('../../lib/utils');
+
+const granuleSuccess = require('../data/granule_success.json');
+const granuleFailure = require('../data/granule_failed.json');
 
 let fakeExecution;
 let stepFunctionsStub;
 let testCumulusMessage;
 
+const mockedFileSize = 12345;
+
+const granuleFileToRecord = (granuleFile) => {
+  const granuleRecord = {
+    size: mockedFileSize,
+    ...granuleFile,
+    key: aws.parseS3Uri(granuleFile.filename).Key,
+    fileName: granuleFile.name,
+    checksum: granuleFile.checksum
+  };
+
+  if (granuleFile.path) {
+    granuleRecord.source = `https://07f1bfba.ngrok.io/granules/${granuleFile.name}`;
```

Comment:
Curious why this is here?

Reply:
I naively copied it from the place where these tests were originally defined: https://github.com/nasa/cumulus/pull/1180/files#diff-1193811283527188dc17f7fc0fda5130L40

Many of these new tests in the model test files were defined in `test-es-indexer.js`, but they were really testing the models so I moved them. I didn't make much effort to refactor them for the sake of time, but I can take a look at this

Reply:
It seems like this host comes from the provider definition in the fixture that is used in some of the tests:

https://github.com/nasa/cumulus/blob/master/packages/api/tests/data/granule_success.json#L77